autoload/neomake/makers/ft/ruby.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#ruby#EnabledMakers()
-    return ['mri', 'rubocop', 'reek']
+    return ['mri', 'rubocop', 'reek', 'rubylint']
```

Comment:
@wsdjeg 
The "executable" check is done in https://github.com/neomake/neomake/blob/1b585580f2f1d33dc115ebf86466bbb4112ed8da/autoload/neomake/utils.vim#L97-L107 (via `neomake#utils#MakerIsAvailable`) already.  But this should probably use Vim's `executable` however.
Creating a PR for that.  In this PR here it's fine.